graph/src/components/ethereum/adapter.rs

```diff
@@ -955,7 +955,7 @@ pub async fn blocks_with_triggers(
 
     // Make sure `to` is included, even if empty.
     block_hashes.insert(to_hash);
-    triggers_by_block.entry(to).or_insert(Vec::new());
```

Comment:
`Vec::new` is a const fn, so I wouldn't change this.

Reply:
Makes sense! 

Reversed it